.gitignore

```diff
@@ -4,3 +4,4 @@ mediapipe/MediaPipe.tulsiproj/*.tulsiconf-user
 mediapipe/provisioning_profile.mobileprovision
 .configure.bazelrc
 .user.bazelrc
+.vs
```

Comment:
Please, remove this change. (It's ok to have it locally, but then just don't commit).